### ISSUE TITLE
 fields inside a nested super table are not saved

### ISSUE THREAD

**Issue by Kotstar1** [engram-design/SuperTable#374]
**Description**

After I updated Super Table to a new Version I saw that I was able to select a Super Table field inside a Matrix Block that is already inside a Super Table field. Is that a Bug or an intended feature? Because, when I Saved, the fields inside the last Super Table were not saved.

Field Layout: Matrix -> SuperTable -> Matrix -> SuperTable -> Textfield <-- is not saved

**Steps to reproduce**
1. Create a Matrix
2. Create a SuperTable inside the Matrix
3. Create a Matrix inside the SuperTable
4. Create a SuperTable inside the Matrix
5. Create a field inside the SuperTable 
6. Save

**Additional info**
- Plugin version: 2.5.4
- Craft version: 3.5.1
- Multi-site: No

**Comment by engram-design** [comment]
This is correct behaviour as despite it being a valid field option to pick, it is not possible to have Matrix -> SuperTable -> Matrix -> SuperTable field setups.

I'll see what I can do to put back this restriction, but it was changed due to a good cleanup that we needed to do to prevent code duplication and special handling with Matrix fields nested in Super Table fields.

Just keep a mental note not to go crazy with nesting, as you have been until now.
